File: tests/unittests/codegen/snapshots/python/basic.py
from datetime import date, datetime, time
from decimal import Decimal
from uuid import UUID


class OperationNameResultLazy:
    something: bool


class OperationNameResult:
    id: str
    integer: int
    float: float
    boolean: bool
    uuid: UUID
    date: date
    datetime: datetime
    time: time
    decimal: Decimal
    lazy: OperationNameResultLazy

File: tests/unittests/codegen/snapshots/typescript/basic.ts
type OperationNameResultLazy = {
    something: boolean
}

type OperationNameResult = {
    id: string
    integer: number
    float: number
    boolean: boolean
    uuid: string
    date: string
    datetime: string
    time: string
    decimal: string
    lazy: OperationNameResultLazy
}
